vpr/src/place/place.cpp

```diff
@@ -171,6 +171,11 @@ static const float cross_count[50] = {/* [0..49] */ 1.0, 1.0, 1.0, 1.0828,
                                       2.5610, 2.5864, 2.6117, 2.6371, 2.6625, 2.6887, 2.7148, 2.7410, 2.7671,
                                       2.7933};
 
+double cong_matrix[400][400];
+double cong_matrix_new[400][400];
+
+float congestion_tradeoff = 1.0;
```

Comment:
I got the same thing as Soheil. Here is my suggestion:
1) Remove the current command line option that you added and use place_chan_width instead.
2) Make this congestion_tradeoff a command line option with value between [0,1] and give the user control to change it if they want instead of a global variable in place.cpp.